packages/terra-demographics-banner/src/DemographicsBanner.scss

```diff
@@ -1,103 +1,87 @@
 @import '~terra-mixins';
 @import './variables';
 
-.terra-DemographicsBanner {
-  background: $demographics-background;
-  color: $demographics-foreground;
-  margin-bottom: 0.714em;
-  padding: 0.714em;
-
-  @media screen and (min-width: 768px) {
-    display: flex;
-    flex-direction: row;
-  }
-}
-
-.terra-DemographicsBanner-value {
-  display: inline-flex;
-
-  @if $terra-bidi {
-    [dir='ltr'] & {
-      margin-right: 0.357em;
-
-      &:last-child {
-        margin-right: 0;
-      }
-    }
-
-    [dir='rtl'] & {
-      margin-left: 0.357em;
-
-      &:last-child {
-        margin-left: 0;
-      }
+/* stylelint-disable selector-class-pattern  */
+
+:local {
+  .demographics-banner {
+    background: $demographics-background;
+    color: $demographics-foreground;
+    margin-bottom: 0.714em;
+    padding: 0.714em;
+
+    @media screen and (min-width: 768px) {
+      display: flex;
+      flex-direction: row;
     }
   }
-  @else {
+
+  .value {
+    display: inline-flex;
     margin-right: 0.357em;
 
     &:last-child {
       margin-right: 0;
     }
   }
-}
 
-.terra-DemographicsBanner-value-label {
-  @include terra-margin-end(0.357em);
-}
+  .value-label {
+    @include terra-margin-end(0.357em);
```

Comment:
The postcss-rtl plugin will now take the place of a lot of these mixins.  This can be rewritten as 
```
margin-right: 0.357em;
````